hastory-data/src/Data/Hastory/Types.hs

```diff
@@ -41,22 +47,15 @@ Entry
 instance NFData Entry
 
 instance Hashable Entry where
```

Comment:
This should be generatable now. 
I think you can just leave out the implementation.
Or just leave it like this, doesn't really matter.